frontend/app/routes/_gcweb-app.update-info.tsx

```diff
@@ -48,21 +48,24 @@ export async function action({ request }: ActionFunctionArgs) {
     phoneNumber: z.string().refine(isPhoneNumber, { message: 'Invalid phone number' }),
   });
 
-  const formData = Object.fromEntries(await request.formData()) as UserInfo;
-  const validFormData = formDataSchema.safeParse(formData);
+  const formData = Object.fromEntries(await request.formData());
+  const parseResult = formDataSchema.safeParse(formData);
 
-  if (!validFormData.success) {
+  if (!parseResult.success) {
     return json({
-      errors: validFormData.error.flatten(),
+      errors: parseResult.error.flatten(),
```

Comment:
https://github.com/colinhacks/zod?tab=readme-ov-file#error-formatting

```suggestion
      errors: parseResult.error.format(),
```

Reply:
[Fixed](https://github.com/DTS-STN/canadian-dental-care-plan/pull/82/files#diff-2bec7e9aba6317cf91db5121aa046967350d305eba20a20d32dfa45459862afbR20)